ENG-FUNDAMENTALS-CHECKLIST.md

```diff
@@ -93,3 +93,13 @@ More details on [Retrospectives](agile-development/retrospectives/readme.md)
 - [ ] Ensure feedback is detailed and repeatable
 
 More details on [Engineering Feedback](engineering-feedback/readme.md)
+
+## Developer Experience (DevEx)
```

Comment:
I would suggest rooting this in the essential tasks presented in the root devex readme. I want to avoid people getting the impression that if they have a dev container they have a good dev experience. While a solid dev container implementation can promote good devex, the dev container's effectiveness should be measured on whether or not it enables the following capabilities. I have seen plenty that do not hit these marks. Additionally, for projects that are unable to leverage dev containers the essential tasks still provide targets that apply.

On a local environment the following can be done 
1) Build/Compile source to verify it is free of syntax errors and compiles
2) Execute all automated tests (unit, e2e, etc)
3) Start/Launch end-to-end to simulate execution in a deployed environment.
4) Attach debugger to started solution or running automated tests, set breakpoints, step through code, and inspect variables.

Additionally, the efun survey questions for dev ex also ask about these subjects.

Reply:
@charleszipp ok, just to make sure, do you want me to remove the checkboxes I wrote and instead add the 4 points you mentioned here for local environment? 
Or you meant to remove the dev containers one, and add the 4 points to the existing list?

Reply:
Please do remove the dev containers one. You could probably add your point as a note below that dev containers to steer people towards using dev containers as a commonly used way to satisfy the checklist.

I would suggest starting with the points I provided then feel free to add to those anything you feel should be part of that checklist. So basically if you want to take a stab at merging the two (minus the dev container  one) that would be good. I would be interested in seeing what you come up with there.

Hopefully that makes sense. Feel free to reach out directly if you need more clarification.

Reply:
Thanks for the clarification @charleszipp ,
I updated the list and now it contains your 4 points + additional 2 I felt were good fit for a checklist.
Please let me know wdyt.